src/time_zone_info.cc

```diff
@@ -738,7 +813,12 @@ bool TimeZoneInfo::Load(const std::string& name) {
   auto zip = cctz_extension::zone_info_source_factory(
       name, [](const std::string& n) -> std::unique_ptr<ZoneInfoSource> {
         if (auto z = FileZoneInfoSource::Open(n)) return z;
-        if (auto z = AndroidZoneInfoSource::Open(n)) return z;
+        if (kIsAndroid) {
+          if (auto z = AndroidZoneInfoSource::Open(n)) return z;
+        }
+        if (kIsFuchsia) {
+          if (auto z = FuchsiaZoneInfoSource::Open(n)) return z;
+        }
```

Comment:
What was the logic behind making these conditional?  Maybe I asked that before?  I don't recall.